README.md

```diff
@@ -95,16 +91,16 @@ defineCustomElementReact({
 })
 ```
 
-### Utiliser le nouveau custom element
+### Using the new custom element
 
-Dans notre premier exemple, le composant React s'appelait `AnyReactComponent` (en pascal case).
-Le custom element doit répondre aux normes de l'HTML et aux bonnes pratiques en vigueur, donc avec un namespace et en pascal case.
+In our previous example, the React component was called `AnyReactComponent` (in Pascal case). 
+The custom element must comply with HTML standards and best practices, so it should have a namespace and be in Pascal case.
```

Comment:
You mean kebab case no ?  

kebab-case: owc-any-react-component

PascalCase: OwcAnyReactComponent

Reply:
thanks ! done